packages/sfui/frameworks/react/components/SfListItem/SfListItem.tsx

```diff
@@ -29,7 +29,7 @@ const SfListItem = polymorphicForwardRef<typeof defaultListItemTag, SfListItemPr
     <Tag
       ref={ref}
       className={classNames(
-        'inline-flex items-center gap-2 w-full hover:bg-neutral-100 active:bg-neutral-200 cursor-pointer',
+        'inline-flex items-center gap-2 w-full hover:bg-neutral-100 active:bg-neutral-200 cursor-pointer focus-visible:outline focus-visible:outline-offset focus-visible:relative focus-visible:z-10',
```

Comment:
I added `z-index` on focus to make the ring position over other list items. There was an issue where focus ring got covered by the background color of the list item below when it was hovered. Let me know if you have any concerns about this solution.

Reply:
@dkacper I think the outline focus should be rounded, according to designs.

Reply:
No, I talked to Jacek and he said focus ring must fit component's shape. Most of our components has rounded corners, but not in this case. It might be misleading, so we may ask Jacek to extend guidelines in this regard.